azurerm/resource_arm_container_group.go

```diff
@@ -256,9 +303,19 @@ func flattenContainerGroupContainers(containers *[]containerinstance.Container)
 		}
 
 		if len(*container.Ports) > 0 {
-			containerConfig["port"] = *(*container.Ports)[0].Port
+			containerPort := *(*container.Ports)[0].Port
+			containerConfig["port"] = containerPort
+			// protocol isn't returned in container config, have to search in container group ports
+			protocol := ""
+			for _, cgPort := range *containerGroupPorts {
```

Comment:
do we need a check to ensure this isn't nil?

Reply:
Done.